todo_app/app.py

```diff
@@ -1,15 +1,22 @@
-from flask import Flask
+from flask import Flask, render_template, redirect, url_for
 
 from todo_app.flask_config import Config
+from todo_app.data.session_items import get_items, add_item
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
 
-@app.route('/')
+@app.route('/', methods=['GET','POST'])
 def index():
-    return 'Hello World!'
-
+    items = get_items()
+    return render_template('index.html',items=items)
 
+@app.route('/new', methods=['GET','POST'])
+def new():
+    add_item('item')
+    item = request.form.get('submit')
```

Comment:
This isn't accessing the field passed in currently - that field is labelled as `name` in the form, so you'll want to access it with that

It's worth noting also that the add_item() function is currently being called with a hardcoded value here, so we would ignore the form value